bld010/byob
# Write states/senators seed file

#### What's this PR do?

Modified states table to include name, seed data is updated to include full name, senators table now includes columns for rank, party, url, and twitter_account.

Also adds updated basic schema xml file to repo.

#### Where should the reviewer start?

Check latest migration file, then review seeds/state.js

#### How should this be manually tested?

Run latest migration and seed in postgres, verify states have full names, and senators have a value for all columns (twitter may be null)

#### Any background context you want to provide?
n/a

#### What are the relevant tickets?

[Include rank, party, url, twitter for senators](https://app.clubhouse.io/briannadelvalle/story/35/as-a-user-i-want-to-see-the-rank-party-url-and-twitter-handle-of-a-senator-in-order-to-understand-who-they-are-and-how-to)
[Update schema](https://app.clubhouse.io/briannadelvalle/story/25/as-a-dev-i-want-to-update-my-states-table-schema-to-reflect-my-updated-data-structure)
[Create seed files](https://app.clubhouse.io/briannadelvalle/story/22/as-a-developer-i-want-to-create-seed-files-so-that-i-can-verify-data-will-appear-appropriately-in-the-state-and-senators)

#### Screenshots (if appropriate)

#### Questions:
